Library/Formula/cracklib.rb

```diff
@@ -22,5 +25,17 @@ def install
                           "--without-python",
                           "--with-default-dict=#{HOMEBREW_PREFIX}/share/cracklib-words"
     system "make", "install"
+
+    resource("cracklib-words").stage do |r|
+      share.install "cracklib-words-#{r.version}" => "cracklib-words"
+    end
+  end
+
+  def post_install
+    system "#{bin}/cracklib-packer < #{share}/cracklib-words"
```

Comment:
You can probably do `pipe_output("#{bin}/cracklib-packer", (share/"cracklib-words").read, 0)`.